src/styles/blocks/_clock.scss

```
@@ -0,0 +1,112 @@
+.clock {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  margin-left: -150px;
+  margin-top: -200px;
+
+  .main-timer {
+    width: 300px;
+    height: 300px;
+    background-color: #000;
+    border: 5px solid goldenrod;
+    border-radius: 150px;
+    box-shadow: 6px 6px 15px 0 #848484;
+  }
+  
+  .hour-hand,
+  .hour-hand::before,
+  .minute-hand {
+    position: absolute;
+    width: 10px;
+    height: 10px;
+    left: 50%;
+    top: 50%;
+    margin-left: -5px;
+    margin-top: -5px;
+    z-index: 2;
+  }
+
+  .main-timer .hour-hand {
+    width: 20px;
+    height: 20px;
+    z-index: 3;
+    background-color: #fff;
+    border-radius: 20px;
+    margin-left: -10px;
+    margin-top: -10px;
+
+    animation: watch-circle 60s linear infinite forwards;
+
+  }
```

```suggestion
    animation: watch-circle 60s linear infinite forwards;
  }
```
no empty lines in the end of the block, fix in other places